Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec bin/rails s
+job: bundle exec sidekiq
```

Comment:
I'm torn. _Yes totally_, but, those are IMO more system level, and unless we really really want to standardize on using `brew services` (or whatever) and get them to run in the foreground so that exiting foreman also exists them, they don't fit into how I approach using Foreman.